VAMobile/src/utils/remoteConfig.ts

```diff
@@ -59,6 +61,7 @@ export const defaults: FeatureToggleValues = {
   decisionLettersWaygate: true,
   patientCheckIn: false,
   patientCheckInWaygate: true,
+  claimPhaseExpansion: false,
```

Comment:
I put claimPhaseExpansion last in my PR so that will probably conflict with these. Though probably we should alphabetize them at this point, which would help avoid conflicts too

Reply:
alphabetized